worldedit-core/doctools/src/main/kotlin/com/sk89q/worldedit/internal/util/DocumentationPlatform.kt

```diff
@@ -55,11 +58,20 @@ class DocumentationPlatform : AbstractPlatform() {
 
     override fun matchWorld(world: World?) = error("Documentation does not provide this")
 
-    override fun registerCommands(commandManager: CommandManager?) = error("Documentation does not provide this")
+    override fun registerCommands(commandManager: CommandManager?) {}
 
-    override fun setGameHooksEnabled(enabled: Boolean) = error("Documentation does not provide this")
+    override fun setGameHooksEnabled(enabled: Boolean) {}
 
-    override fun getCapabilities() = error("Documentation does not provide this")
+    override fun getCapabilities(): MutableMap<Capability, Preference> {
+        val capabilities: MutableMap<Capability, Preference> = EnumMap(Capability::class.java)
+        capabilities[Capability.CONFIGURATION] = Preference.NORMAL
```

Comment:
I don't think it really matters but I think all of these should be `PREFER_OTHERS`